### ISSUE TITLE
Grails 3.2 Resource Bundle does not display text with correct encoding

### ISSUE THREAD

**Issue by apandichi** [grails/grails-core#10832]
I'm going to copy paste the details of this issue out of this StackOverflow question: 
https://stackoverflow.com/questions/44365852/grails-3-2-resource-bundle-does-not-display-text-with-correct-encoding
I have the same issue and could not find any indication of this being reported before as an issue (or answered on SO)

_Since Grails 3.2.8 (and above) the resource bundle within my application does not work as expected. German umlauts (or general UTF-8 chars that are not part from ASCII) are display with ��. E.g. the German word über is displayed as ��ber._

_The source file is clearly encoding using UTF-8._

An important detail that is not mentioned in the above SO question is the fact that the issue does not happen when running the grails app with "grails run-app ...", it happens when packaging the app as a standalone jar/war.

### Steps to Reproduce

1. Add some special characters to a message in messages.properties
2. Use the message in a GSP
3. Package the app as a standalone jar/war
4. Run the Grails app with "java -jar" 

### Expected Behaviour

The special characters should be displayed correctly.

### Actual Behaviour

Special characters are displayed as question marks: ��

### Environment Information

- **Operating System**: Ubuntu 16.04.3 LTS
- **Grails Version:** 3.2.9
- **JDK Version:** 1.8.0_101-b13

**Comment by jeffbrown** [comment]
> 2. Use the message in a GSP

Does the problem occur when you do something like `<g:message code="some.message.with.special.characters"/>`?

**Comment by apandichi** [comment]
@jeffbrown Yes, it does.

**Comment by saw303** [comment]
I've noticed that this issue only applies when I build my war using `gradlew clean build` while having my Grails project open in IntelliJ. If I close IntelliJ before building with Gradle everything works as expected. Strange behaviour....
--

**Comment by apandichi** [comment]
I don't think IntelliJ has anything to do with it. I have the build under a CI server setup (TeamCity, to be fair...), and I still have the problem.

**Comment by sdelamo** [comment]
@apandichi could you upload a sample project?

**Comment by apandichi** [comment]
@sdelamo After doing some more digging, it looks like the culprit was a missing "-Dfile.encoding=UTF-8" in the Gradle build command line. 😅 Thank you for your time!

**Comment by saw303** [comment]
@apandichi can you provide a small snipped of that Gradle build file ?

**Comment by apandichi** [comment]
It's not the Gradle build file that was wrong, it was the command that I was using.
Previously it was: `./gradlew assemble`
Now it's: `./gradlew -Dfile.encoding=UTF-8 assemble`

Now for the "why didn't this happen before?" -- I recently disabled Gradle daemons on the CI server. Probably the Gradle daemon had the encoding property already setup, so I suspect the encoding broke after I disabled the Gradle daemon. But that's only a speculation, I haven't really tested out this theory.
